lib/cff/model.rb

```diff
@@ -1,5 +1,3 @@
-# frozen_string_literal: true
```

Comment:
Not sure whether this is required and won't have any unintended side effects.

Reply:
Good point! Removing this